locust/test/test_main.py

```diff
@@ -854,9 +1265,22 @@ def test_web_options(self):
                 stdout=PIPE,
                 stderr=PIPE,
             )
-            gevent.sleep(1)
-            self.assertEqual(200, requests.get("http://127.0.0.1:%i/" % port, timeout=3).status_code)
-            proc.terminate()
+            try:
+
+                def server_ready():
+                    try:
+                        response = requests.get(f"http://127.0.0.1:{port}/", timeout=0.1)
+                        return response.status_code == 200
+                    except requests.RequestException:
+                        return False
+
+                poll_until(server_ready, timeout=5, poll_interval=0.1)
```

Comment:
Here as well, I think the default poll_interval is 0.1, so we shouldn't need to pass this argument